lib/FreeRTOS-Plus-TCP/source/FreeRTOS_DNS.c

```diff
@@ -813,21 +814,16 @@ uint8_t *pucUDPPayloadBuffer;
 size_t xPlayloadBufferLength;
 DNSMessage_t *pxDNSMessageHeader;
 
-	xPlayloadBufferLength = pxNetworkBuffer->xDataLength - sizeof( UDPPacket_t );
-	if ( xPlayloadBufferLength < sizeof( DNSMessage_t ) )
-	{
-		return pdFAIL;
-	}
+	/* In this context, 'pxNetworkBuffer->xDataLength' is the
```

Comment:
Agreed, although I don't think we should remove the check for minimal size of a DNS message.